README.md

```diff
@@ -1,80 +1,56 @@
-# graphite-nozzle
+# paas-cf-apps-statsd
 
-This library consumes events off the Cloud Foundry Firehose, processes them, and then sends them off to a StatsD endpoint. Note that it is still being developed and shouldn't be considered production-ready.
+This application consumes container metrics off the Cloud Foundry Doppler daemon, processes them based on provided metrics template, and then sends them off to a StatsD endpoint.
+CPU, RAM and disk usage metrics for app containers will be sent through to StatsD as a Gauge metric. It will get metrics for all applications provided user has access to. Note that it is still being developed and shouldn't be considered production-ready.
 
 ## Getting Started
 
-* A user who has access to the Cloud Foundry Firehose configured in
-your manifest
-
-```
-properties:
-  uaa:
-    clients:
-      graphite-nozzle:
-        access-token-validity: 1209600
-        authorized-grant-types: authorization_code,client_credentials,refresh_token
-        override: true
-        secret: <password>
-        scope: openid,oauth.approvals,doppler.firehose
-        authorities: oauth.login,doppler.firehose
-
-```
-
-* A Graphite and StatsD server (see [here](https://github.com/CloudCredo/graphite-statsd-boshrelease) for a Graphite/StatsD BOSH release).
-* Golang installed and configured (see [here](https://golang.org/doc/install) for a tutorial on how to do this).
-* godep (see [here](https://github.com/tools/godep) for installation instructions).
-
-Once you've met all the prerequisites, you'll need to download the library and install the dependencies:
-
-```
-mkdir -p $GOPATH/src/github.com/pivotal-cf
-cd $GOPATH/src/github.com/pivotal-cf
-git clone git@github.com:pivotal-cf/graphite-nozzle.git
-cd graphite-nozzle
-godep restore
-godep go build
-```
-
-Finally, run the app:
-
+* Clone this repository
+* Deploy the app to CloudFoundry.You can use application flags or envirnoment variables to configure it.
```

Comment:
That's my auto-completion suggesting wrong thing. I always have to use space to suppress it.